mlflow/R/mlflow/inst/DOCKERFILE_TEMPLATE

```diff
@@ -0,0 +1,41 @@
+# Build an image that can serve mlflow models.
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND='noninteractive'
+ENV MLFLOW_HOME='/opt/mlflow'
+RUN apt-get -y update && apt-get install -y --no-install-recommends \
+         wget \
+         curl \
+         nginx \
+         ca-certificates \
+         bzip2 \
+         build-essential \
+         cmake \
+         openjdk-8-jdk \
+         git-core \
+         maven \
+    && rm -rf /var/lib/apt/lists/*
+
+# Download and setup miniconda
+RUN curl -L https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh >> miniconda.sh
+RUN bash ./miniconda.sh -b -p /miniconda; rm ./miniconda.sh;
+ENV PATH="/miniconda/bin:$PATH"
+ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+ENV GUNICORN_CMD_ARGS='--timeout 60 -k gevent'
+# Set up the program in the image
+WORKDIR /opt/mlflow
+
+{install_mlflow}
+
+# Install the latest stable version of R
```

Comment:
Have you considered building on top of rocker/r-ver to abstract away some of this?

Reply:
Good suggestion! I'm certainly consider this as an option now as it looks promising, although at the moment I see rocker/r-ver will use 'debian:buster' as its base image and also will install 'default-jdk' for some reason. These could be things that make installing `mlflow` trickier.

Reply:
I see. Maybe that's better then. In any case, we could maybe borrow some ideas from there, in particular for dependency management, using RSPM will greatly speed up build times.